WebNetSample/Program.cs

```diff
@@ -39,8 +42,17 @@
 
 app.UseAuthorization();
 
-app.MapControllerRoute(
-    name: "default",
-    pattern: "{controller=Home}/{action=Index}/{id?}");
+app.UseEndpoints(options =>
+{
+    options.MapControllerRoute(
+                name: "images",
```

Comment:
Guess this tabulation is too big, let's make the code consistent with _default_ route pattern